src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -80,6 +97,7 @@ public IEnumerable<(string manifestId, string? informationalPath, Func<Stream> o
 
         public IEnumerable<string> GetManifestDirectories()
```

Comment:
implementation side, we just need to make sure GetManifestDirectories and GetManifests is the only method used to search for manifest. Or other caller might get the old behavior

Reply:
Agreed, I think that's the intention regardless of this change.